apps/wallet-mobile/src/features/Swap/common/helpers.ts

```diff
@@ -99,3 +99,11 @@ export const sortTokensByName = (a: Balance.TokenInfo, b: Balance.TokenInfo, wal
 
   return nameA.localeCompare(nameB, undefined, {sensitivity: 'base'})
 }
+
+export const switchDayAndMonth = (dateString: string): string => {
```

Comment:
hey @SorinC6 is there a ticket for this one? cuz this should be based on the `Intl` I don't think we should update it.

Reply:
please ping Product.

Reply:
If we can tell the formatter to use 2 digits for years, that may be done. But changing the order of the months and days or whatever... that's locale based, not design based.

Reply:
@jorbuedo not sure how to achieve it without switching the order 🤔

Reply:
can you check it @SorinC6 ?